bookmarks/styles/variables-dark.scss

```
@@ -24,5 +24,9 @@ $link-color-light: $link-color !default;
 $alternative-color: #59bdb9;
 $alternative-color-dark: #73f1eb;
 
+$green-color: #8aaa7c;
+$green-color-dark: #4a912b;
```

```suggestion
$secondary-link-color: #8aaa7c;
```
`$green-color-dark` doesn't seem to be used. Since `$green-color` is just another variant of `$link-color`, maybe rename it to `$secondary-link-color`.

Let's also move this up to the other link color variables.

---

Did changes as requested, doing other tasks before pushing changes.